### ISSUE TITLE
new: no-invalid-schema

### ISSUE THREAD

**Issue by aladdin-add** [not-an-aardvark/eslint-plugin-eslint-plugin#61]
// invalid
```js
// https://github.com/eslint/eslint/blob/master/tests/fixtures/testers/rule-tester/no-invalid-schema.js
module.exports.schema = [
    {
        "enum": []
    }
];
```

// valid
```js
module.exports.schema = [
    {
        "enum": ["foo", "bar"]
    }
];
```

**Comment by not-an-aardvark** [comment]
Is `enum: []` actually an invalid schema, or is it just a schema that's impossible to satisfy?
